SDWebImage/SDWebImageCoderHelper.m

```diff
@@ -0,0 +1,255 @@
+/*
+ * This file is part of the SDWebImage package.
+ * (c) Olivier Poitrey <rs@dailymotion.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+#import "SDWebImageCoderHelper.h"
+#import "SDWebImageFrame.h"
+#import "UIImage+MultiFormat.h"
+#import "NSImage+WebCache.h"
+#import <ImageIO/ImageIO.h>
+
+@implementation SDWebImageCoderHelper
+
++ (UIImage *)animatedImageWithFrames:(NSArray<SDWebImageFrame *> *)frames {
```

Comment:
Copied from previous `SDWebImageWebPCoder` code. Use frame class instead of two arrays. :)